omnisharptest/omnisharpJestTests/optionChangeObserver.test.ts

```diff
@@ -0,0 +1,164 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { timeout } from 'rxjs/operators';
+import { from as observableFrom, Subject, BehaviorSubject } from 'rxjs';
+import { Options } from '../../src/shared/options';
+import { registerOmnisharpOptionChanges } from '../../src/omnisharp/omnisharpOptionChanges';
+
+import * as jestLib from '@jest/globals';
```

Comment:
It is possible to globally import jest type definitions so you can just use 'describe' - however the functions will conflict with mocha ones.  So for now we manually import the type definitions.

We also need to name it something not 'jest' as it will throw errors because jest (the runner) is already imported.